Core/src/org/sleuthkit/autopsy/contentviewers/ContactArtifactViewer.java

```diff
@@ -231,6 +311,16 @@ public boolean isSupported(BlackboardArtifact artifact) {
         return artifact.getArtifactTypeID() == BlackboardArtifact.ARTIFACT_TYPE.TSK_CONTACT.getTypeID();
     }
 
+    /**
+     * Clear any artifact specific state.
+     */
+    private void reset() {
```

Comment:
Brian was looking for one resetComponent method that would reset the state of all of the components.